build.sbt

```diff
@@ -5,14 +5,14 @@ val catsTestkitScalaTestVersion = "2.1.5"
 val catsVersion                 = "2.9.0"
 val catsScalacheckVersion       = "0.3.1"
 val catsTimeVersion             = "0.3.4"
-val circeVersion                = "0.14.3"
```

Comment:
Let's wait for `0.14.5`?   Version `0.14.4` triggers a problem with SSO.

https://github.com/circe/circe/issues/2089